repartition scenarios docs (#108)
loadimpact/k6-docs: src/data/markdown/docs/01 guides/02 Using k6/14 Scenarios.md

hideFromSidebar: false
---

> ### 🎉 New in v0.27.0
>

> > This feature is new as of version 0.27.0. Usage of this feature is optional and for the vast majority,

Reviewer: ```suggestion
> This feature is new as of version 0.27.0. Usage of this feature is optional and for the vast majority of users,
```

> existing scripts and configurations will continue to work as before. For a list of breaking changes,

> > see the [k6 v0.27.0 release notes](https://github.com/loadimpact/k6/releases/tag/v0.27.0)).

Reviewer: ```suggestion
> see the [k6 v0.27.0 release notes](https://github.com/loadimpact/k6/releases/tag/v0.27.0).
```

> ## Benefits

Reviewer: I don't think this should be a header, since we haven't explained what scenarios are at this point. The paragraph below is both an introduction to the feature and a list of its benefits, so I think we're fine with just the preamble of "There are several benefits...".

Scenarios allow us to make in-depth configurations to how VUs and iterations are scheduled, allowing

> us to model diverse traffic patterns in our load tests. There are several benefits of using scenarios in your tests:

Reviewer: - "Scenarios allow us..." and "allowing us" sounds repetitive.
- Nit: the voice of the first sentence is 1st person plural, but the second one switches to 2nd person, which is a bit jarring. I understand we want to make the docs personable, but in many cases keeping an indirect voice is more approachable in technical documentation IMO, and pronouns are messy nowadays 😅 

I would rewrite this to:
```suggestion
k6 scenarios enable configuring how VUs and iterations are scheduled, making it possible to model diverse traffic patterns in load tests. There are several benefits of using scenarios:
```

Author: Makes total sense. Thanks!

</div>

> ## Common options

Reviewer 2: `Common options` at this moment looks out of scope.

A possible solution could be to introduce earlier `Executors` and next `Common options`. 

For example, we point that each `executor` has its own configuration options but there are a set of common options shared between all the executor types, and describe the `Common` options section.

Author: Switched places of the sections. Good catch!

> ## Common options

Reviewer: This should still be an `h3` as it's part of the Configuration section.

Author: While you're correct, I think I'd like to keep it an h2 for now, as I want it to be displayed in the sidebar for smoother navigation. 🙇‍♂️
